pages/common/conan-frogarian.md

```diff
@@ -0,0 +1,8 @@
+# conan frogarian
+
+> Displays the conan frogarian.
+> More information: <https://docs.conan.io/>.
+
+- Display the conan frogarian:
```

Comment:
@sbrl good place to ask: do we have a preference as to which verb to use: "Print", "Display", "Output", "Report", etc.?  I like "print" the best.  WDTY?

Reply:
I think I prefer `Display`, so probably fine to leave it as is. 👍🏻